lib/readable_streambuffer.js

```diff
@@ -42,6 +42,9 @@ var ReadableStreamBuffer = module.exports = function(opts) {
     if (sendMore) {
       sendData.timeout = setTimeout(sendData, frequency);
     }
+    else {
+      sendData.timeout = null;
```

Comment:
No, it doesn't call it directly indeed but by clearing `sendData.timeout` the check in the `this._read` function will.